src/main/java/club/sk1er/patcher/util/chat/ChatHandler.java

```diff
@@ -168,7 +168,7 @@ public static void appendMessageCounter(IChatComponent chatComponent, boolean re
                     } else {
                         entry.messageCount++;
                         entry.lastSeenMessageMillis = currentTime;
-                        chatComponent.appendSibling(new ChatComponentIgnored(ChatColor.GRAY + " (" + decimalFormat.format(entry.messageCount) + ")"));
+                        chatComponent.appendSibling(new ChatComponentIgnored(ChatColor.GRAY + " [" + "x" + decimalFormat.format(entry.messageCount) + "]"));
```

Comment:
Since this is a personal choice, I won't merge this. If you introduced a config option that defaulted to the old style, that would be fine.

Reply:
I don't think it's worth the trouble to have a config for this, so I'll also drop this change; nothing really wrong with the original format.